.github/workflows/update-artifacts.yml

```diff
@@ -0,0 +1,54 @@
+name:  Update artifacts manifest
+
+on:
+  push:
+    branches:
+      -  main
+
+jobs:
+    update-artifacts:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Checkout Code
+          uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
+
+        - name: Setup Kubectl
+          id: install-kubectl
+          uses: azure/setup-kubectl@v3
+          with:
+            version: 'latest'
+
+        - name: Setup Helm
+          uses: azure/setup-helm@v4.2.0
+          with:
+            version: v3.15.1
+
+        - uses: actions/setup-python@v5
+          with:
+            python-version: '3.10'
+
+        - name: Install python dependencies
+          run: pip install -r scripts/artifacts/requirements.txt
+
+        # Generate artifacts manifest for inframold charts
+        - name: Generate Artifacts Manifest for Each Chart
+          run: |
+            charts_list=$(ls charts | grep inframold)
```

Comment:
lets not do this rather. this can be error prone. rather use the static values for inframold